packages/ui-react/src/components/Player/Player.tsx

```diff
@@ -72,7 +72,14 @@ const Player: React.FC<Props> = ({
   const handleComplete = useEventCallback(onComplete);
   const handleUserActive = useEventCallback(onUserActive);
   const handleUserInactive = useEventCallback(onUserInActive);
-  const handleFirstFrame = useEventCallback(onFirstFrame);
+  const handleFirstFrame = useEventCallback(() => {
+    // a11y: auto focus to play/pause button
+    const playPauseButton = document.querySelector('.jw-controlbar [aria-label="Play"], .jw-button-container [aria-label="Pause"]') as HTMLDivElement | null;
```

Comment:
Opening the trailer while on the inline player page will target the wrong play/pause button (the first one found in the DOM).

Using the player container will ensure the correct player is queried:

```suggestion
    const playPauseButton = playerRef.current?.getContainer().querySelector('.jw-controlbar [aria-label="Play"], .jw-button-container [aria-label="Pause"]') as HTMLDivElement | null;
```

Reply:
Thanks. That's a side-case I haven't thought about.